### PR TITLE
Printer configs

### PR DESCRIPTION
**Pull request by SteveGotthardt**
Config: for Voxelab Aquia with v1.0.0 (X1, etc.) and v1.0.3 boards with HC32F460

These are updated with the company specific choices for the hc32f460 Kconfig

Signed-off-by: Steven Gotthardt <gotthardt@gmail.com>

### CODE REVIEW

**Comment by JamesH1978** [comment]
Thanks for submitting these configs, can you please refer to https://github.com/Klipper3d/klipper/blob/master/docs/Example_Configs.md point 4 and add the regression tests.

Thanks
James

File: config/printer-voxelab-aquila-v1.0.0-2020.cfg [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# This file contains pin mappings for the Voxelab Aquila with
+# HC32F460 on the X1_Main_Board_V1.0.0 mainboard (X1, more).
+# To use this config:
+# during: "make menuconfig" select the the HC32F460.
+# The V1.0.0 board uses 0x8000 for the Flash Application Address
```
Comment (JamesH1978) [inline config/printer-voxelab-aquila-v1.0.0-2020.cfg:5]: Not sure if these 2 directions wont be confusing, as they are not options in the make menuconfig. 

Thanks
James

File: config/printer-voxelab-aquila-v1.0.3-2021.cfg [hunk 2]
```diff
@@ -0,0 +1,157 @@
+# This file contains pin mappings for the Voxelab Aquila with
+# HC32F460 on the 1.0.3 mainboard. Chassis sticker 'H32'
+To use this config, during
+# "make menuconfig" select the the HC32F460.
+# The V1.0.3 board uses 0xC000 for the Flash Application Address
```
Comment (JamesH1978) [inline config/printer-voxelab-aquila-v1.0.3-2021.cfg:5]: Same comment on these 2 directions as per my other comment

Thanks
James

**Review by JamesH1978 — COMMENTED** [review]
Just my comments on the flash directions

Thanks
James

**Comment by JamesH1978** [comment]
Ah...just read the comments properly and your other PR, makes more sense now.

If those options are added, then i can approve this.

Thanks
James

**Comment by flicker812** [comment]
Good day,
Do you have or are you working on a config for Sovol SV06+?

Thank you,
Lee

**Comment by KevinOConnor** [comment]
What's the status of this PR?  Seems like it would be useful to add the examples, but as James points out earlier, we'd like them to follow the "example config guidelines".

-Kevin

**Comment by SteveGotthardt** [comment]
I took a look at the guidelines. I was trying to run the check locally. Is that possible? or do I have to push and then wait? or?

thanks

**Comment by KevinOConnor** [comment]
There is information on running the regression tests at: https://www.klipper3d.org/Debugging.html#running-the-regression-tests

-Kevin

**Comment by SteveGotthardt** [comment]
Thanks. I'll use that

**Comment by Sineos** [comment]
Any update @SteveGotthardt?

**Comment by SteveGotthardt** [comment]
@Sineos 
I have been working on my house. I will update this issue soon. Thanks for your patience ;)
S

**Pull request closed without merging**